pages/AddSurveyQuestionCommonUtils.js

```diff
@@ -0,0 +1,58 @@
+const { expect } = require("@playwright/test");
+
+export async function reset(page){
+    await page.getByRole('button', { name: 'Reset', exact: true }).click();
+    await expect(page.getByRole('heading', { name: 'Reset Confirmation' })).toBeVisible();
+    await expect(page.getByLabel('Reset Confirmation')).toContainText('Are you sure you want to reset? Once the reset is done, the data cannot be retrieved back.');
+    await expect(page.getByRole('button', { name: 'Cancel' })).toBeEnabled();
+    await expect(page.getByRole('button', { name: 'Reset' })).toBeEnabled();
+    await page.getByRole('button', { name: 'Reset' }).click();
+  }
+
+export async function questionFieldsCharacterLimit(page){
+  await page.getByPlaceholder('Enter question code').fill('Q11111111111111111111111111111111111111111111111111111111111112');
+  await expect(page.locator("(//p[normalize-space()='63/64'])")).toBeVisible();
```

Comment:
What is this `"(//p[normalize-space()='63/64'])"`?

Reply:
Inorder for me to use undo (ctrl+z), I had to fill 1 charcter less than the max limit. That's the reason I used that css selector.